mozilla/fx-private-relay
# MPP-3442: Email template for users receiving their first forwarded email

This PR fixes MPP-3442.

# New feature description
See the figma file which has the designs for both the mobile and web layouts.

https://www.figma.com/file/VWDfo6NG52RbGmWGfuKX2F/Improved-Onboarding-for-Free-Relay-Users?type=design&node-id=803-4899&mode=design&t=huPFNpZ09yOZdiNz-0

# Screenshot (if applicable)

Demo:

https://github.com/mozilla/fx-private-relay/assets/59676643/72256483-0db2-4d32-9a18-5f33fab57bd4

**Demo in Gmail/Outlook:**
(Note: Removed arrow pointing towards the dashboard button, and gradient border for "What you can do with Firefox Relay" does not get rendered on Outlook).
https://github.com/mozilla/fx-private-relay/assets/59676643/362316e4-6579-4546-8d93-0c20478c7d83


# How to test
1. [Set up end-to-end local email](https://github.com/mozilla/fx-private-relay/blob/main/docs/end-to-end-local-email-dev.md) or push branch to dev server
2. Sign in and go to http://127.0.0.1:8000/api/v1/docs/#/first-forwarded-email
3. Expand the endpoint
4. Click "Try it out"
5. Enter a generated masks email as the value to "mask" in the request body.
6. Click "Execute"
7. Observe that the email was sent, and the design matches the files in figma.

It should show a 201 response code
It should send the first forwarded email

# Checklist (Definition of Done)
